docker/picard/Dockerfile

```diff
@@ -7,4 +7,19 @@ LABEL maintainer="Jishu Xu <jishuxu@broadinstitute.org>" \
   website="http://broadinstitute.github.io/picard"
 
 ENV version 2.10.10
```

Comment:
I actually would like to move this "ENV" just before the installation of Picard.

Reply:
Thanks! I will move it.